plugins/outputs/logzio/README.md

```diff
@@ -14,12 +14,11 @@ This plugin sends metrics to Logz.io over HTTPs.
   
   ## Timeout for HTTP requests
```

Comment:
nitpick: spacing

Reply:
@glinton You mean to put token/url/timeout together in one block? or everything as one block? not sure what about the spacing

Reply:
Hmm, I must be losing my mind. I thought I saw the tabbing was off.
```toml
    ## Timeout for HTTP requests
  # timeout = "5s"
```
But that's clearly not the case

Reply:
![image](https://user-images.githubusercontent.com/2653109/58645592-ed6a2900-82c0-11e9-8462-0e57d3ca5dca.png)
Had to make sure I'm still sane :smile: the raw file looks good though.

Reply:
@glinton hahah thank you again for the PR :)